THM-ATLAS/vue-frontend
# Add register form, fix create user modal in admin panel

To test:

* Create new user in admin panel and try to login
* Logout, press register and create new user
* Login with user upon success 

This PR also fixes/improves some minor things, see files changed for what exactly
